main.py

```diff
@@ -63,7 +63,7 @@ async def on_ready():
     """Call when discord bot's status is ready
     """
     print("起動しました")
-    await bot.add_cog(event.BotEvents())
+    await bot.add_cog(event.BotEvents(bot))

```

Comment:
Sorry, I can't find line that using `bot` variable.
Would you know where it is used?

Reply:
oh...umm...i...don't know....too...

Reply:
If it's working, All right!!